packages/vue/src/auth/mapAuthState.ts

```diff
@@ -216,7 +216,7 @@ export const useFrontegg = () => {
 
   const loginWithRedirect = () => {
     const path = fronteggAuth.getCurrentRoute();
-    if (!path.startsWith(authState.routes.hostedLoginRedirectUrl ?? defaultFronteggRoutes.hostedLoginRedirectUrl)) {
+    if (!path.startsWith(authState.routes.hostedLoginRedirectUrl ?? defaultFronteggRoutes.hostedLoginRedirectUrl ?? "oauth/callback")) {
```

Comment:
what's this?

Reply:
this is the default route, fix lint of nullable check